### ISSUE TITLE
cannot find module pino

### ISSUE THREAD

**Issue by xenoterracide** [new10com/axios-logger#2]
```
Error: Cannot find module 'pino'
Require stack:
- /Users/calebcushing/IdeaProjects/briggo/services/node_modules/@new10com/axios-logger/dist/src/logger/logger.js
- /Users/calebcushing/IdeaProjects/briggo/services/node_modules/@new10com/axios-logger/dist/src/axios-logger.js
- /Users/calebcushing/IdeaProjects/briggo/services/app-lib/graph/packages/app/src/shared/logging/LoggerConfig.ts
- /Users/calebcushing/IdeaProjects/briggo/services/app-lib/graph/packages/app/src/context/ContentfulConfig.ts
- /Users/calebcushing/IdeaProjects/briggo/services/app-lib/graph/packages/app/src/ApolloServerConfig.ts
- /Users/calebcushing/IdeaProjects/briggo/services/app-lib/graph/packages/app/src/App.ts
- /Users/calebcushing/IdeaProjects/briggo/services/app-lib/graph/packages/app/src/index.ts
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/Users/calebcushing/IdeaProjects/briggo/services/node_modules/@new10com/axios-logger/src/logger/logger.ts:1:1)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
```

using yarn 2 I added the following to the yarnrc, this adds dependencies to the package, and then the module is working fine.

```yaml
packageExtensions:
  '@new10com/axios-logger@^0.1':
    dependencies:
        pino: '^6'
        pino-pretty: '^4'
```

**Comment by azakordonets** [comment]
Thanks, will check where pino is still used, because i think i removed it since we didn't need it anymore.

**Comment by azakordonets** [comment]
I had a problem with publishing new versions of the library to npm from Github. Now it should be fixed in v0.1.3 - can you please give it a try ? @xenoterracide

**Comment by xenoterracide** [comment]
looks good
